activity.py

```diff
@@ -291,13 +284,12 @@ def _one_to_one_connection(self, tp_channel):
             return
         bus_name, connection, channel = json.loads(tp_channel)
         logger.debug('GOT XMPP: %s %s %s', bus_name, connection, channel)
-        Connection(bus_name, connection, ready_handler=lambda conn:
-                   self._one_to_one_connection_ready_cb(
-                       bus_name, channel, conn))
+        conn = TelepathyGLib.Connection.new(bus_name, connection)
```

Comment:
Hey @pro-panda, should this call have a dbus daemon as first argument?  Is so in a later call around line 770.  My test didn't hit these lines.